src/components/ClusterOverview/TopConsumers/TopConsumers.js

```diff
@@ -14,83 +12,141 @@ import {
   DashboardCardTitleHelp,
 } from '../../Dashboard/DashboardCard';
 import { ClusterOverviewContextGenericConsumer } from '../ClusterOverviewContext';
+import { InlineLoading } from '../../Loading';
+
+const ConsumersBar = ({ now, description, label }) => (
+  <Row className="kubevirt-consumers__bar">
+    <Col lg={4} md={4} sm={4} xs={4}>
+      <OverlayTrigger
+        overlay={<Tooltip id={`tooltip-for-${description}`}>{description}</Tooltip>}
+        placement="top"
+        trigger={['hover', 'focus']}
+        rootClose={false}
+      >
+        <div className="kubevirt-consumers__bar-description">{description}</div>
+      </OverlayTrigger>
+    </Col>
+    <Col lg={5} md={5} sm={5} xs={5} className="kubevirt-consumers__bar-column-progress">
+      <div className="progress kubevirt-consumers__bar-progress">
+        <ProgressBar min={0} max={100} now={now} key={1} isChild />
+        <ProgressBar min={0} max={100} now={100 - now} key={2} bsClass="progress-bar progress-bar-remaining" isChild />
+      </div>
+    </Col>
+    <Col lg={3} md={3} sm={3} xs={3} className="kubevirt-consumers__bar-label">
+      <OverlayTrigger
+        overlay={<Tooltip id={`tooltip-for-${label}`}>{label}</Tooltip>}
+        placement="top"
+        trigger={['hover', 'focus']}
+        rootClose={false}
+      >
+        <span>{label}</span>
+      </OverlayTrigger>
+    </Col>
+  </Row>
+);
 
-const sortConsumers = (metrics, filter, sortBy) => {
-  const metricKey = Object.keys(metrics).find(key => metrics[key].title === sortBy);
-  const filteredConsumers =
-    filter !== 'All' ? metrics[metricKey].consumers.filter(c => c.kind === filter) : metrics[metricKey].consumers;
-  const max = Math.max(...filteredConsumers.map(c => c.usage));
-  return filteredConsumers
+ConsumersBar.propTypes = {
+  now: PropTypes.number.isRequired,
+  description: PropTypes.string.isRequired,
+  label: PropTypes.string.isRequired,
+};
+
+const getConsumers = metric => {
+  const max = Math.max(...metric.consumers.map(c => c.usage));
```

Comment:
now value will be always relative to the highest consumers, which I don't think is very useful.

Reply:
after discussion with @jelkosz this is what we want. The first consumer bar is 100% and others are relative to the first one.

Reply:
I talked to @jelkosz and we concluded that it would be good to include some icon with a popover that would explain what is happening. E.g. `something something, The value xy is relative to the current higest value (100% bar)`. It could also include something about the query.

Reply:
created a followup https://github.com/kubevirt/web-ui-components/issues/313 - I will ask UXD for help